lib/routes/account.js

```diff
@@ -15,6 +15,7 @@ var MAX_ACTIVE_SESSIONS = 200
 var butil = require('../crypto/butil')
 var userAgent = require('../userAgent')
 var requestHelper = require('../routes/utils/request_helper')
+var geodb = require('fxa-geodb')()
```

Comment:
IIUC, the @jrgm and @jbuck are going to download the maxmind db outside of the normal installation process. When including the geodb, a path to the database should be passed, where the path is set in configuration.